src/ontology/Makefile

```diff
@@ -46,8 +49,7 @@ ANNOTATE_ONTOLOGY_VERSION = annotate -V $(ONTBASE)/releases/$(VERSION)/$@ --anno
 OTHER_SRC =                 $(PATTERNDIR)/definitions.owl 
 ONTOLOGYTERMS =             $(TMPDIR)/ontologyterms.txt
 PATTERNDIR=                 ../patterns
-DOSDP_SCHEMA=                http:// # change to PURL when ready.
-PATTERN_TESTER=              simple_pattern_tester.py
+PATTERN_TESTER=              dosdp validate -i
```

Comment:
I think this might need to be `$(DOSDPT) validate -i`? Otherwise it says `dosdp: not found`
But the version of dosdp in my current ODK Docker image doesn't seem to have a validate command, so this isn't working for me anyway - `Command not found: validate`.

(when I run `sh run.sh make dosdp_validation -B`)

Reply:
oops yeah, sorry, forgot about that! Can you try

```
sh run.sh make PATTERN_TESTER="simple_pattern_tester.py" dosdp_validation -B
```
 in the meantime?

Reply:
(you will need the new ODK image which is not available yet for the dosdp command)

Reply:
yeah, that runs the existing pattern tester as expected